test/rng_testsuite/conformance_tests/minstd_rand_rand0_test.pass.cpp

```diff
@@ -27,12 +27,17 @@
 //     The 10000th consecutive invocation of a default-constructed object of type minstd_rand
 //     produces the value 399268537
 
+#include <iostream>
 
+#if ONEDPL_USE_DPCPP_BACKEND
 #include "common_for_conformance_tests.hpp"
 #include <oneapi/dpl/random>
+#endif // ONEDPL_USE_DPCPP_BACKEND
```

Comment:
I see that if` `ONEDPL_USE_DPCPP_BACKEND` is not defined, the test is built even if we use a compiler with SYCL. In default way the test is skipped even with SYCL Compiler. Is it expected?

Reply:
If ONEDPL_USE_DPCPP_BACKEND is not defined the test is built without RNG part and just skipped during execution. 
The same situation if ONEDPL_USE_DPCPP_BACKEND == 0.

If ONEDPL_USE_DPCPP_BACKEND is defined (as not a 0) the test is executed with RNG functionality

So my understanding is that the test is executed with RNG functionality only in case of SYCL compiler

Reply:
I'm not sure I understand expectations.
So, the default behavior of the test is to be skipped even with SYCL compiler, but in test system in case of SYCL compiler you manually write ONEDPL_USE_DPCPP_BACKEND=1?

Reply:
Our expectations are next - tests are skipped with non-SYCL compilers
And executed with SYCL one

You suggested to change previous internal macro to public one
Could you please tell - where this public macro is defined? 

We do not change and/or define ONEDPL_USE_DPCPP_BACKEND by ourselves

Reply:
ONEDPL_USE_DPCPP_BACKEND is set to 0 in CMakeLists if ONEDPL_BACKEND is TBB: https://github.com/oneapi-src/oneDPL/blob/main/CMakeLists.txt#L106

Reply:
The macro is defined by user.
Actually I suggested 2 different ways. That's why I asked about purpose.
So, as I mentioned here (https://github.com/oneapi-src/oneDPL/pull/75#discussion_r555729961, 2nd suggestion) it's better to use Compiler macros if we want to distinguish different compilers, not oneDPL backends. Seems, the 2nd suggestion is more applicable here.

P.S. The name and description of PR are confusing since those speak about SYCL backend, not compiler.

Reply:
ONEDPL_USE_DPCPP_BACKEND is described [here](https://github.com/oneapi-src/oneDPL/blob/main/documentation/library_guide/pstl/macros.rst):


> This macro enables the use of the DPC++ policies. **_When the macro is not defined (by default) or evaluates to non-zero, DPC++ policies are enabled._** When the macro is set to 0 there is no dependency on the oneAPI DPC++/C++ Compiler and runtime libraries. Trying to use DPC++ policies will lead to compilation errors.

I believe @capatober refers to this statement:

> When the macro is not defined (by default) or evaluates to non-zero, DPC++ policies are enabled.

It says that _undefined_ is treated as true.

Reply:
Now I got why we have this misunderstanding - will change PR name to the right one
@AlexVeprev could you please tell - do we have macro which could be used for non-SYCL compilers skipping? 

Or should we add such as Andrey suggested (2nd suggestion)?